python/ray/tests/test_numba.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from numba import njit
+import pytest
+import numpy as np
+import unittest
+import ray
+
+
+@njit(fastmath=True)
+def centroid(x, y):
+    return ((x / x.sum()) * y).sum()
+
+
+# A Wrapper to call centroid func
+@ray.remote
+def centroid_wrapper(x, y):
+    return centroid(x, y)
+
+
+class NumbaTest(unittest.TestCase):
+    def setUp(self):
+        ray.init()
+
+    def tearDown(self):
+        ray.shutdown()
+
+    def assertEqualHelper(self, func, expected, *args):
```

Comment:
Let's get rid of this helper, since we don't use this kind of helper in any of our other tests and I think it makes the test a bit harder to read.